test/react-native/features/app_data.feature

```diff
@@ -0,0 +1,81 @@
+Feature: App data
```

Comment:
I think it would be worth adding a check on `the event "unhandled" is` in each scenario.